trigger.js

```diff
@@ -105,10 +108,30 @@ class TriggerHappy {
         this._executeTriggers(triggers);
     }
 
-    _onUpdateJournal(journal) {
-        if (journal.id === this._journalId || journal.name === this.journalName)
+    _onUpdateJournal(journal, update) {
+        if (update._id === this._journalId || journal.name === this.journalName)

```

Comment:
update.name may not be defined (usually isn't) so it is actually no better a test than journal.name. In the case that journal is defined (i.e. for the user doing the change it will match). For other clients the name match will never work - until the fix is released

Reply:
Yeah, `update.name` isn't defined, but the purpose of having is for when you create a new entry which gets "Journal Entry" name, then you change the name.
If I keep listening to the same id (which would be null), then you wouldn't load triggers when you rename your journal to match the appropriate journal name, so checking for update.name is important in that case.